tests/test_no_extra_install.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests Sklearn GradientBoostingClassifier converters.
+"""
+import unittest
+import warnings
+
+import numpy as np
+
+from hummingbird.ml._utils import sklearn_installed, lightgbm_installed, xgboost_installed
+
+
+class TestNoExtra(unittest.TestCase):
+    """
+    These tests are meant to be run on a clean container after doing
+    `pip install hummingbird-ml` without any of the `extra` packages
+    """
+
+    # Test no LGBM returns false on lightgbm_installed()
+    def test_lightgbm_installed_false(self):
+        warnings.filterwarnings("ignore")
+        assert not lightgbm_installed()
+
+    # Test no XGB returns false on xgboost_installed()
+    def test_xgboost_installed_false(self):
+        warnings.filterwarnings("ignore")
+        assert not xgboost_installed()
+
+    def test_import_convert_no_extra(self):
```

Comment:
Can you please add a comment on this? Basically you should be able to still do import even if you don't have any extra, right?

Reply:
done